app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,46 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Expense Transactions app</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
```

Comment:
- [OPTIONAL] Although this is not a requirement in the project requirements but I suggest that you implement a sign out link in the navbar on all pages in your project. this would be a very nice addition to your project UI.